torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4992,6 +5012,11 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                SkipInfo('TestCommon', 'test_variant_consistency_eager'),
            ),
            sample_inputs_func=sample_inputs_addbmm),
+    OpInfo('attn',
+           dtypes=floating_types_and(torch.bfloat16),
+           dtypesIfCUDA=floating_types_and(torch.float16, *[torch.bfloat16] if CUDA11OrLater else []),
```

Comment:
Nice fix.
We should fix that for future bootcampers!

Reply:
out of curiosity, what needs to be fixed here? i cargo-culted this line from other ops.